tensorboard/backend/event_processing/event_file_loader.py

```diff
@@ -49,8 +51,12 @@ def Load(self):
     tf.logging.debug('Loading events from %s', self._file_path)
     while True:
       try:
-        with tf.errors.raise_exception_on_not_ok_status() as status:
-          self._reader.GetNext(status)
+        if len(inspect.getargspec(self._reader.GetNext).args) is 0:
```

Comment:
Also looks like pylint is complaining about getargspec being deprecated on py3.6+.  For now probably fine to just suppress `deprecated-method` warning on this line.  I think getfullargspec() is the replacement, but it's not available on Python 2...

Reply:
Thanks yeah - I saw that - I also assumed the code works fine (with the "is 0" check) since it didn't complain about this.

Reply:
I think it didn't complain because right now with existing tf-nightly len(args) == 2 so this branch isn't triggered.  But when tf-nightly picks up your change, len(args) will be == 1 (since "self" will still count as an argument), so this check will still fail, even though the intent is that it should pass.  So I think this should be `if len(args) == 1: # only the "self" argument` to anticipate that change.